web/server/codechecker_server/api/product_server.py

```diff
@@ -602,7 +604,9 @@ def removeProduct(self, product_id):
         with DBSession(self.__session) as session:
             product = session.query(Product).get(product_id)
             if product is None:
-                LOG.erorr("Product with ID %d does not exist!", product_id)
+                msg = "Product with ID {0} does not exist!".format(product_id)
```

Comment:
The same question here as before.
The typo fix was not enough?

Reply:
Same as above, `msg` was undefined.